tests/main/helpers/test_search_helpers.py

```diff
@@ -247,42 +246,44 @@ def test_build_search_query_unknown_lot_is_dropped(self):
             'lot': 'saasaas',
         })
 
-        assert search_helpers.build_search_query(self.g6_framework, request.args, self.lot_filters, self._loader(),
-                                                 self._lots_by_slug) == {'index': 'g-cloud-6'}
+        assert search_helpers.build_search_query(request.args, self.lot_filters, self._loader(),
+                                                 self._lots_by_slug) == {}
 
     def test_build_search_query_multiple_lots_are_all_dropped(self):
         request = self._request({
             'lot': 'saas,paas',
         })
 
-        assert search_helpers.build_search_query(self.g6_framework, request.args, self.lot_filters, self._loader(),
-                                                 self._lots_by_slug) == {'index': 'g-cloud-6'}
+        assert search_helpers.build_search_query(request.args, self.lot_filters, self._loader(),
+                                                 self._lots_by_slug) == {}
 
     def test_build_search_query_no_keywords_drops_q_parameter(self):
         request = self._request({
             'q': '',
         })
 
-        assert search_helpers.build_search_query(self.g6_framework, request.args, self.lot_filters, self._loader(),
-                                                 self._lots_by_slug) == {'index': 'g-cloud-6'}
+        assert search_helpers.build_search_query(request.args, self.lot_filters, self._loader(),
+                                                 self._lots_by_slug) == {}
 
     def test_build_search_query_no_page_drops_page_parameter(self):
         request = self._request({
-            'index': 'g-cloud-6',
             'page': '',
         })
 
-        assert search_helpers.build_search_query(self.g6_framework, request.args, self.lot_filters, self._loader(),
-                                                 self._lots_by_slug) == {'index': 'g-cloud-6'}
+        assert search_helpers.build_search_query(request.args, self.lot_filters, self._loader(),
+                                                 self._lots_by_slug) == {}
 
     def test_build_search_query_g5_dots_id_search_replaces_dots_with_dashes(self):
         request = self._request({
-            'index': 'g-cloud-6',
             'q': 'some text 5.G4.1005.001',
         })
 
-        assert search_helpers.build_search_query(self.g6_framework, request.args, self.lot_filters, self._loader(),
+        assert search_helpers.build_search_query(request.args, self.lot_filters, self._loader(),
                                                  self._lots_by_slug) == {
-            'index': 'g-cloud-6',
             'q': 'some text 5-G4-1005-001',
         }
+
+
+def find_search_summary(res_data):
```

Comment:
Should this helper be in this module? Or in `tests.helpers`?

Reply:
Ah yes, good spot. I think I thought it was search helpers to help with testing. Brain fail.

I'll move it into tests.helpers.BaseApplicationTest